core/search/fields.py

```diff
@@ -306,3 +309,12 @@ def get_condition(self, expression: Expression, remainder: List[str]) -> Any:
             low, high = self._get_date_range(expression)
 
         return and_(self.column >= low, self.column < high)
+
+
+class RelationField(BaseField):
+    accepts_subquery = True
+
+    def get_condition(self, expression: Expression, remainder: List[str]) -> Any:
+        if not isinstance(expression, Subquery):
+            raise UnsupportedGrammarException("Only Field Group is allowed for Relation Field")
```

Comment:
User usually doesn't know what is FieldGroup - it should be called just 'subquery'
```suggestion
            raise UnsupportedGrammarException("Only subquery is allowed for relation field")
```